src/Components/Onboarding/Components/OnboardingLoadingCollection.tsx

```diff
@@ -2,7 +2,13 @@ import { Flex, Spacer, Spinner, Text } from "@artsy/palette"
 import { FC } from "react"
 import { useOnboardingFadeTransition } from "../Hooks/useOnboardingFadeTransition"
 
-export const OnboardingLoadingCollection: FC = () => {
+interface OnboardingLoadingCollectionProps {
+  type: string
```

Comment:
Can make this optional by adding in a `?`: `type?: string`, then no need for `type=""` up above